### PR TITLE
Add description

### PR DESCRIPTION
**Pull request by karakayasemi**
Necessary descriptions are added. See #3

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
     </two-factor-providers>
 
     <dependencies>
-        <owncloud min-version="9.1" max-version="10.0" />
+        <owncloud min-version="10.0" max-version="10.0.99" />
```
Comment (DeepDiver1975) [inline appinfo/info.xml]: max 10.0 please - see my comment in security app

**Comment by karakayasemi** [comment]
my fault this commit should not be in here. Version is corrected at #5

**Comment by karakayasemi** [comment]
@DeepDiver1975 merge?

**Comment by DeepDiver1975** [comment]
I'd like to have some native english review on the strings ... 
@tomneedham maybe?

**Comment by karakayasemi** [comment]
@DeepDiver1975 seems everything okay

**Pull request merged**